### ISSUE TITLE
File name(dataset/main_xml1\scene0299_00\immask_8.png,.....) does not exist

### ISSUE THREAD

**Issue by rjhuang27** [lzqsd/InverseRenderingOfIndoorScene#4]
Hi, I encountered the following problems
![image](https://user-images.githubusercontent.com/69796850/128830436-5d92500f-d650-4c39-a124-09b868190442.png)
this problem is caused by this
![image](https://user-images.githubusercontent.com/69796850/128827129-06f55d1f-6641-43d2-aff2-14518741bff1.png)
I have downloaded OpenRooms dataset  and put them in the ./dataset folder, there are no pictures of  png format in main_xml1 folder. Now, I don't know where are the pictures of png format from.

**Comment by lzqsd** [comment]
Hi, did you also download the Mask.zip from OpenRooms dataset and put them together with the image?  :)

**Comment by rjhuang27** [comment]
Thank you,but I only have the link of image.zip and demo.zip, I can't download Material, Geometry, Mask.zip and so on. Could you share them, thanks again!

**Comment by lzqsd** [comment]
you are welcome! Did you send an email to openroomsdataset@gmail.com? If yes, you should have received the download links to other zip files already.

**Comment by rjhuang27** [comment]
Yes, I have sent, but I only received the links of image.zip and demo.zip. Other zip files haven't download links. My email is rjhuang27@gmail.com, Could you share them, Thanks!

**Comment by lzqsd** [comment]
From my gmail record, I can see that I sent the links to the rjhuang27@gmail.com on Sat, Aug 7, 1:32 PM (3 days ago). I will send you the links again now :). Please let me know if you do not receive it. Thanks!

**Comment by rjhuang27** [comment]
Thank you. Now the download link is available.
